### PR TITLE
Update LICENSE to Hippocratic 2.1

### LINKED ISSUE #3: Update LICENSE to Hippocratic 2.1
To discourage use of Lansing Codes' community-building software by anyone who would violate others' human rights, we are adopting the latest version of the ethical source license, Hippocratic 2.1.

In order to use the latest version of this license, the following changes are required:

1. Add or update the `LICENSE` file in this project to the [Hippocratic 2.1 Markdown](https://firstdonoharm.dev/version/2/1/license.md) text and make the following substitutions:
    - Replace `[SOFTWARE NAME]` with the GitHub organization and project name (e.g. `@lansingcodes/webfont`)
    - Replace `(YEAR)` with `2019-Present`
    - Replace `(COPYRIGHT HOLDER(S)/AUTHOR(S))` with `Humanity Codes, LLC`
2. Add or update the `README.md` file in this project to include the following Markdown snippet:
    ``` md
    ## License
    
    [Hippocratic 2.1](https://firstdonoharm.dev)
    
    Copyright (c) 2019-Present, Humanity Codes LLC
    ```
3. If a `package.json` file exists in this project, add or update the `license` field in the file to look like this:
    ``` json
    "license": "Hippocratic-2.1",
    ```

---

**When submitting the pull request to resolve this issue, please assign @egillespie as the reviewer.**

### PR DESCRIPTION
**Pull request by dhcrain**
Resolves #3

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+[SOFTWARE NAME] Copyright (YEAR) (COPYRIGHT HOLDER(S)/AUTHOR(S))(“Licensor”)
```
Comment (egillespie) [inline LICENSE:1]: @dhcrain, do you mind substituting `[SOFTWARE NAME]`, `(YEAR)`, and `(COPYRIGHT HOLDER(S)/AUTHOR(S))` to match the values in #3?

**Review by egillespie — CHANGES_REQUESTED** [review]
@dhcrain, I requested one small change to the `LICENSE` file. 🙂

File: LICENSE (hunk 1 above)
Comment (dhcrain) [reply LICENSE]: Ahh yes, I definitely got on autopilot there.

**Review by egillespie — APPROVED** [review]
Nailed it. Nice work! That's four isn't it? 😁 🎉 👏

**Pull request merged**